Rip out all the chrome.app & background page usage.  The app APIs aren't
available to us, so switch to tabs instead.  We just don't need the background
page logic at all.

--- a/mosh_app/mosh_client.js
+++ b/mosh_app/mosh_client.js
@@ -21,6 +21,7 @@ var kSSHDefaultPort = 22;
 var kMoshDefaultPort = 60001;
 
 function onUpdateAvailable() {
+  return;
   chrome.runtime.getBackgroundPage(function(bg) {
     var updateDiv = document.querySelector('#update');
     if (bg.state.updateAvailable != null) {
@@ -98,6 +99,7 @@ window.onload = function() {
   var mainTable = document.querySelector('#main-table');
   var height = mainTable.scrollHeight + (margin * 2);
   var width = mainTable.scrollWidth + (margin * 2);
+return;
   var bounds = chrome.app.window.current().innerBounds;
   bounds.setSize(width, height);
   bounds.setMinimumSize(width, height);
@@ -237,6 +239,8 @@ function onConnectClick(e) {
   var id = 'mosh_window_' + args['mode'] + '_' + args['user'] + '@' +
       args['addr'] + ':' + args['port'];
 
+  document.location.replace(`/plugin/mosh/mosh_window.html?args=${btoa(JSON.stringify(args))}`);
+  return;
   chrome.runtime.getBackgroundPage(function(bg) {
     while (id in bg.state.windows) {
       // ID already exists. Keep adding underscores until it doesn't.
@@ -301,24 +278,15 @@ function updateMode(e) {
 }
 
 function onSshKeyClick(e) {
-  chrome.app.window.create('ssh_key.html', {
-    'bounds': {
-      'width': 400,
-      'height': 300,
-    },
-    'id': 'ssh_key',
-  });
+  chrome.tabs.create({url: '/plugin/mosh/ssh_key.html', active: true});
   // Prevent default handling.
   return true;
 }
 
 function onPrefsClick(e) {
-  chrome.app.window.create('mosh_prefs.html', {
-    'bounds': {
-      'width': 400,
-      'height': 300,
-    },
-    'id': 'preferences_editor',
+  chrome.tabs.create({
+    url: '/html/nassh_preferences_editor.html?profileId=mosh',
+    active: true,
   });
   // Prevent default handling.
   return true;
--- a/mosh_app/mosh_window.js
+++ b/mosh_app/mosh_window.js
@@ -43,12 +45,14 @@ function execMosh() {
 
   // Workaround to return focus to terminal on fullscreen.
   // See https://code.google.com/p/chromium/issues/detail?id=402340
+/*
   var appWindow = chrome.app.window.current();
   appWindow.onFullscreened.addListener(function() {
     appWindow.hide();
     appWindow.show();
     terminal.focus();
   });
+*/
 
   document.title += ' - ' + window.args['addr'];
 
